File: 프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.java
import java.io.*;

class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        String s = br.readLine();
        int a = Integer.parseInt(s.split(" ")[0]);
        int b = Integer.parseInt(s.split(" ")[1]);

        for (int i = 0; i < b; i++) {
            for (int j = 0; j < a; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: Kotlin/프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.kt
import java.io.*

fun main(args: Array<String>) {
	val br = BufferedReader(InputStreamReader(System.`in`))
	val s = br.readLine()
	
	val (a, b) = s.split(' ').map { it.toInt() }

	for (i in 1..b) {
		for (j in 1..a) {
			print("*")
		}
		println()
	}
}
